CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ option(UR_BUILD_ADAPTER_CUDA "Build the CUDA adapter" OFF)
 option(UR_BUILD_ADAPTER_HIP "Build the HIP adapter" OFF)
 option(UR_BUILD_ADAPTER_NATIVE_CPU "Build the Native-CPU adapter" OFF)
 option(UR_BUILD_ADAPTER_ALL "Build all currently supported adapters" OFF)
+option(UR_BUILD_ADAPTER_L0_V2 "Build the (experimental) Level-Zero v2 adapter" OFF)
```

Comment:
Not sure if we should build `UR_BUILD_ADAPTER_L0_V2` if `UR_BUILD_ADAPTER_ALL` is set - it is pretty quite far from fully functional at this moment.

Reply:
No, let's leave it off.